### ISSUE TITLE
Corrupted Downloaded songs from jiosaavn

### ISSUE THREAD

**Issue by Brajo280603** [KRTirtho/spotube#1133]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

While jiosaavn is selected as source. downloaded songs seems to be corrupted. Cannot be played on any other player

### Expected Behavior

Downloaded songs should be playable on any player.

### Steps to reproduce

1. Open app
2. Switch audio source to jiosaavn
3. Download any song
4. Play the downloaded file.
5. file found as corrupted

### Operating System

Android

### Spotube version

3.4.0

### Installation source

Website (spotube.netlify.app) or (spotube.krtirtho.dev)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Try redownloading. Make sure your audio player supports mp4 audio. You can also check from Spotube as well.

**Comment by kingsbeep** [comment]
Yes I checked it just now, this problem is very much present even now. Files are corrupted after they're downloaded and won't play on offline music player like Poweramp. Metadata is fine though

**Comment by KRTirtho** [comment]
> Songs that are download in 320kbps (from jiosaavn) are having this issue.

So, when you lower audio quality that doesn't happen?

**Comment by AamiRobin** [comment]
Still, the same issue tried all audio quality.
